.travis.yml

```diff
@@ -15,6 +15,11 @@ matrix:
         apt:
           packages:
             - oracle-java9-installer # Forces use of even newer JDK 9 build
+    - os: linux
+      sudo: false
```

Comment:
Currently, the `group` key below can only be used for `sudo: required`, so this will need to be updated.

Reply:
Oh, cool. Thank you so much for reviewing this!!! :D